.pre-commit-config.yaml

```diff
@@ -14,13 +14,24 @@ repos:
   - id: debug-statements
   - id: forbid-new-submodules
 - repo: https://github.com/psf/black
-  rev: stable
+  rev: 19.10b0
```

Comment:
it's better to have exact versions, as a new version doesn't break black.

Instead you can use ``pre-commit autoupgrade`` to autobump lint versions periodically.